test/box-tap/merger.test.lua

```diff
@@ -765,4 +765,13 @@ for _, input_type in ipairs({'buffer', 'table', 'tuple'}) do
     end
 end
 
+-- merger.new(kd, {}) -- pass zero amount of sources.
```

Comment:
Is it better to add a comment, that test should be analyzed with `-fsanitize=implicit-integer-sign-change` flag set, isn't it?

Reply:
I have added it firstly, but there are many details:

* One should know, where to add the flag (cmake/compiler.cmake), because it is not enabled now.
* One can't just enable the flag and run the test, because other code fails on similar problems.
* It is possible to get rid of the `-fno-sanitize-recover=<...>` flag and look on the output visually, but it is easier to do from the console/a separate script.

When I started to describe 'certain behavior is not important, the test passes before and after the commit, it should just go over certain path to trigger possible ubsan warnings' it looks too vague, as for me. At least, when a reader is not in context of the patch (just reads the test file).

The test file is about various merger cases. It doesn't obligate the cases to fail in a past. So I think it is okay to briefly describe the case itself and leave the description why it is added only in the commit message.